client/src/components/common/applicant-info-1.vue

```diff
@@ -1,14 +1,12 @@
 <template>
   <v-form v-model="isValid" ref="step1" id="applicant-info-1-form">
-    <v-container fluid class="pa-0" id="applicant-info-1">
+    <v-container fluid class="pa-0 mt-5" id="applicant-info-1">
```

Comment:
I changed all whitespace between header and sub-header to mt-5 (so it now looks better and is consistent).